src/main/java/com/extendedclip/deluxemenus/action/ClickActionTask.java

```diff
@@ -131,6 +132,20 @@ public void run() {
                 player.sendMessage(StringUtils.color(executable));
                 break;
 
+            case LOG:
+                String[] logParts = executable.split(" ", 2);
+
+                Level logLevel;
+
+                try {
+                    logLevel = Level.parse(logParts[0].toUpperCase());
+                } catch (IllegalArgumentException e) {
+                    logLevel = Level.INFO;

```

Comment:
add in a debug message in the catch using `plugin.debug()` to inform that the Level is not set correctly